src/components/Book.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ]  Please do not leave unused code as a comment in your codebase. Kindly remove this, It is always a good practice to fix all linter errors instead of disabling them.

Reply:
Thanks, but these seem to be a problem in eslint itself not detecting props right, please refer to this thread https://github.com/yannickcr/eslint-plugin-react/issues/498.